site/en/docs/extensions/mv3/security/index.md

```diff
@@ -34,29 +34,24 @@ security circumventing most [man-in-the-middle attacks][4].
 
 The Chrome browser limits an extension's access to privileges that have been explicitly requested in
 the [manifest][5]. Extensions should minimize their permissions by only registering APIs and
-websites they depend on. Arbitrary code should be kept to a minimum.
+websites they depend on.
 
 Limiting an extensions privileges limits what a potential attacker can exploit.
 
 ### Cross-origin fetch() {: #xhr }
 
-{% Aside 'warning' %}
-`XMLHttpRequest()` is not supported in Service Workers.
-Use its modern replacement, `fetch()`.
-{% endAside %}
-
-An extension can only use `fetch()` or [XMLHttpRequest][6] to get resources from itself and from domains
-specified in the permissions, as both API's use the same [fetch handler][27] in the service worker.
+An extension service worker can only use `fetch()`, or APIs that use the same [fetch handler][27] to get resources from itself and from domains
+specified in the permissions.
```

Comment:
The original wording was correct. This wording is problematic. The only other API that goes through the service worker's `fetch()` handler is `XMLHTTPRequest()`.